public/src/main/java/com/revenuecat/purchases/models/Period.kt

```diff
@@ -25,6 +25,14 @@ data class Period(
      */
     val iso8601: String
 ) : Parcelable {
+
+    companion object Factory {
+        fun create(iso8601: String): Period {
```

Comment:
Minor: I probably would have called tis fromIso8601Duration or something like that, but this is fine